examples/demo.js

```diff
@@ -84,7 +84,7 @@ window.onload = function () {
     $(this).removeClass('selected');
   });
 
-  function displayMessageOnSaveSequence() {
+  function displayMessageOnSaveSequence() { // displays a notification on saving a sequence to the browser
```

Comment:
We don't use continuous tenses in docs.
Use
When a sequence is saved to a browser, notification is displayed

Reply:
I'll make the changes.